packages/libraries/core/src/client/http-client.ts

```diff
@@ -96,7 +96,7 @@ export async function makeFetchCall(
 
   return await asyncRetry(
     async (bail, attempt) => {
-      logger.info(
+      logger?.info(
```

Comment:
console.error(‘request started’)

Reply:
I changed the logger to optional by default to avoid cluttering console log output if someone missed passing a logger.

Reply:
done!